apt/private/starlark_codegen_utils.bzl

```diff
@@ -0,0 +1,45 @@
+"Utilities for generating starlark source code"
```

Comment:
This is pretty cool, I like it being extracted out. I have all of this in [`deb_import.bzl`](https://github.com/jjmaestro/rules_distroless/blob/b0f06d770cb6702179aa7fc994d1d1541cb4e8f7/apt/private/deb_import.bzl) (coming from #96 and #100).

It's all inside the helpers and it makes the code look complicated (the output is nicely formatted Starlark as well :) but it can probably be done much better with this approach.